docs/index.md

```diff
@@ -109,6 +141,7 @@ def edit():
 def logout():
     # Logout of the Cognito User pool and delete the cookies that were set
     # on login.
+    # Revokes the refresh token to not be used again and removes it from the session.
     # No logic is required here as it simply redirects to Cognito.
     pass
 
```

Comment:
For the comments on this file - please could you make the relevant changes in `example/app.py` and `README.md` - appreciate there is some duplication across these, will have to work to reduce that in future :sweat_smile: